CMakeLists.txt

```diff
@@ -33,7 +33,13 @@ if(CMakePythonDistributions_SUPERBUILD)
 
   option(BUILD_VERBOSE "Build reporting additional information (e.g download progress, ...)" ON)
 
-  option(RUN_CMAKE_TEST "Run CMake test suite when built from sources" ON)
+  if("$ENV{CIBUILDWHEEL}")
+    set(CIBW ON)
+  else()
+    set(CIBW OFF)
+  endif()
+
+  option(RUN_CMAKE_TEST "Run CMake test suite when built from sources" "${CIBW}")
```

Comment:
Is there a way to set the `RUN_CMAKE_TEST` option in the corresponding `tool.cibuildwheel` section in `pyproject.toml` ?